src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -0,0 +1,359 @@
+package org.isf.medicalinventory.gui;
+
+import org.isf.utils.jobjects.InventoryState;
+import org.isf.generaldata.MessageBundle;
+import org.isf.utils.jobjects.ModalJFrame;
+
+import com.toedter.calendar.JDateChooser;
+
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+
+import java.awt.BorderLayout;
```

Comment:
The order of the imports is incorrect; please see https://github.com/informatici/openhospital-gui/tree/develop/.ide-settings